mathics/builtin/numbers/calculus.py

```diff
@@ -1484,12 +1484,23 @@ class Series(Builtin):
     <dt>'Series[$f$, {$x$, $x0$, $n$}]'
         <dd>Represents the series expansion around '$x$=$x0$' up to order $n$.
     </dl>
-
-    >> Series[Exp[x],{x,0,2}]
+    For elementary expressions, `Series` returns the explicit power series as a `SeriesData` expression:
+    >> Series[Exp[x], {x,0,2}]
      = 1 + x + 1 / 2 x ^ 2 + O[x] ^ 3
-    >> Series[Exp[x^2],{x,0,2}]
+    >> % // FullForm
+     = SeriesData[x, 0, List[1, 1, Rational[1, 2]], 0, 2, 1]
+    Replacing the variable by a value, the Series will not be evaluated as
+    an expression, but as a SeriesData object:
+    >> s = Series[Exp[x^2],{x,0,2}]
      = 1 + x ^ 2 + O[x] ^ 3
-
+    >> s /. x->4
+     = 1 + 4 ^ 2 + O[4] ^ 3
+    ``Normal`` transforms a ``SeriesData`` expression into a polinomial:
```

Comment:
Right now we use Mathics'  funky formatting here , not RsT. (Hopefully one day this will change.)

So better is to use single quote, not double backtick.

Reply:
polinomial -> polynomial